ssl/ssl_cert.cc

```diff
@@ -135,7 +135,9 @@
 BSSL_NAMESPACE_BEGIN
 
 CERT::CERT(const SSL_X509_METHOD *x509_method_arg)
-    : x509_method(x509_method_arg) {}
+    : x509_method(x509_method_arg) {
+  this->cert_privatekeys.Init(SSL_PKEY_NUM);
+}
```

Comment:
There's no check on the return value here. The call to `Init` could potentially fail, invalidating logic that assumes the size of always `SSL_PKEY_NUM`.